src/clients/cassandra/CassandraConnector.js

```diff
@@ -1,18 +1,80 @@
 'use strict';
 
 const Promise = require('promise');
+const cassandra = require('cassandra-driver');
+const asyncEachLimit = require('async/eachLimit');
 
-/** Execute a batch of mutations
- * @param {Array<{mutation: string, params: Array<string|number|map}>} mutations
+/** @see http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * TODO: ASYNC_BATCH_LIMIT will need to be tuned.
+ * It is best to keep the batch size small (in the order of 10s).
+ * If the batch size is greater than 5k, then the server will issue a warning.
+ */
+const ASYNC_BATCH_LIMIT = process.env.ASYNC_BATCH_LIMIT || 10;
+
+/** Default Cassandra Client Options
```

Comment:
We don't need any of these comments -- they're very useful now when you're implementing the functionality but they'll just add noise/unnecessary detail for future developers who just want to get a brief overview of what's going on. As such, the commit message or pull request text are better places to put these comments.